### ISSUE TITLE
[infra] remove nuget, hail paket

### ISSUE THREAD

**Issue by haf** [SuaveIO/suave#378]
Remove the NuGet.exe that [never](https://github.com/SuaveIO/suave/issues/190) gets fixed and replace it for paket's `pack` call.

[These lines of Rakefile](https://github.com/SuaveIO/suave/blob/master/Rakefile#L121-L137) should be replaced with [the finished code from here](https://github.com/Albacore/albacore/tree/feature/paket-pack).

In particular, [this test should pass](https://github.com/Albacore/albacore/commit/c0d5d74bc7aed00a6b1f8fc447ba87a9bc11d31f#diff-50013d90e4a233dea5611f45e78c2411R70) and you're good to go!

**Comment by jeremyabbott** [comment]
Hello @haf,

I started looking at this. I'm not clear on what needs to happen though. I understand that the Rakefile should be updated to call paket.exe instead of nuget.exe, but I'm not sure what needs to happen with the finished code in the Albacore/paket-pack branch. It hasn't been merged into master. I'm sure I'm missing something.

**Comment by haf** [comment]
Sorry, I expressed myself unclearly. The albacore code isn't finished yet, but the lines in the rakefile should be replaced with the finished code when it's finished. I did a bit of work on it two weeks ago on the paket branch, specifically adding a flag `use_legacy` [here](https://github.com/Albacore/albacore/blob/feature/paket-pack/spec/nugets_pack_spec.rb#L12).

Albacore should replace the call to nuget with a call to paket unless use_legacy is set. And I've added a dependency to `paket` which makes paket available on the PATH, so there's no need to try to find its location in the file system.

**Comment by jeremyabbott** [comment]
Thanks @haf I looked at all the changes in that branch. It makes sense to me, even with my almost non-existent Ruby experience. So should I wait on this until the Albacore changes from paket-pack are merged to master?

**Comment by haf** [comment]
Hi @jeremyabbott,

Well, you could wait and I can ping you or you can take a stab at it. I was hoping to keep it pretty straight forward; generate the metadata file and run paket pack. I'm in a tunnel of work right now and hope to see the light the upcoming weeks, but right now I'm probably not going to finish that branch.

**Comment by jeremyabbott** [comment]
@haf I'll dig into it some more this evening. I'll probably have more questions.

**Comment by haf** [comment]
Thank you. I appreciate you taking an interest =)
